Content.Client/Fax/UI/FaxBoundUi.cs

```diff
@@ -22,12 +30,47 @@ protected override void Open()
         _window.OpenCentered();
 
         _window.OnClose += Close;
+        _window.FileButtonPressed += OnFileButtonPressed;
         _window.CopyButtonPressed += OnCopyButtonPressed;
         _window.SendButtonPressed += OnSendButtonPressed;
         _window.RefreshButtonPressed += OnRefreshButtonPressed;
         _window.PeerSelected += OnPeerSelected;
     }
 
+    private async void OnFileButtonPressed()
+    {
+        if(_dialogIsOpen)
```

Comment:
```suggestion
        if (_dialogIsOpen)
```
(do this of all ifs, not just this one)